.github/workflows/audit.yaml

```diff
@@ -0,0 +1,19 @@
+name: Security audit
+on:
+  schedule:
```

Comment:
I'm guessing this runs every day? Can you add a comment about this?

Re. only `cron` and `workflow_dispatch`; I'm guessing this is intentionally not run on PRs, since this would cause failures unrelated to the PR (for instance: vulnerability DB was updated)? Can you add a comment about that?